pages/_document.tsx

```diff
@@ -33,4 +39,16 @@ export default class MyDocument extends Document {
       sheet.seal()
     }
   }
+
+  render(): JSX.Element {
+    return (
+      <Html lang="en">
```

Comment:
Sine Next 10's i18n routing support we no longer need to override _document to set a lang

Instead you can add a locales setting and a defaultLocale to next.config.js

https://nextjs.org/docs/advanced-features/i18n-routing